tests/layer_tests/pytorch_tests/test_scatter.py

```diff
@@ -213,13 +212,16 @@ def test_scatter_reduce(self, dim, index, src, dtype, inplace, has_out, reduce,
             pytest.skip(
                 "Cannot test reduce parameters with empty indexes due to issues with empty constant tensor or issues with prim::GetAttr str inputs."
             )
+        kwargs = dict(kwargs_to_prepare_input={"dtype": dtype, "out": has_out}, freeze_model=freeze)
+        if reduce == "mean" and dtype in ["int32", "int64"]:
+            # rounding can be different on torch vs ov
+            kwargs["custom_eps"] = 1.
```

Comment:
can we set such threshold for .5 numbers? For other generated numbers, let us have standard threshold.
Or you can generate numbers in a way to avoid .5 numbers in output?

Reply:
Generate such numbers would be tricky, since we calculate mean, we would need to generate numbers non-symmetrically, or have fixed seed